crates/nu-cli/src/completions/dotnu_completions.rs

```diff
@@ -33,6 +33,12 @@ impl Completer for DotNuCompletion {
         let start_with_backquote = prefix_str.starts_with('`');
         let end_with_backquote = prefix_str.ends_with('`');
         let prefix_str = prefix_str.replace('`', "");
+        // e.g. `./`, `..\`
+        let cwd_only = prefix_str.starts_with('.')
+            && prefix_str
+                .chars()
+                .find(|c| *c != '.')
+                .is_some_and(|c| c == SEP);
```

Comment:
@blindFS Will this only look for backslashes on Windows, rather than `./` too? If so, it might be better to use `is_separator`

Reply:
Oh, good catch.